testsuite/features/step_definitions/common_steps.rb

```diff
@@ -647,11 +647,7 @@
   # Preparation of bootstrap script for different types of clients
   client = client_type == 'traditional' ? '--traditional' : ''
   # Use server if proxy is not defined as proxy is not mandatory
-  if target_type.include? 'server' or $proxy.nil?
-    target = $server
-  else
-    target = $proxy
-  end
+  target = target_type.include? 'server' or $proxy.nil? ? $server : $proxy
```

Comment:
I get a syntactic error in this line.  The following is OK:
`target = (target_type.include? 'server') || $proxy.nil? ? $server : $proxy`

Reply:
Rubocop really doesn't make it easy this time ;-) Now we have different complaints:
```
features/step_definitions/common_steps.rb:650:3: C: Style/ConditionalAssignment: Use the return of the conditional for variable assignment and comparison.
  if target_type.include? 'server' || $proxy.nil? ...
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
features/step_definitions/common_steps.rb:650:6: W: Lint/RequireParentheses: Use parentheses in the method call to avoid confusion about precedence.
  if target_type.include? 'server' || $proxy.nil?
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```